python/contactflowhelper/README.md

```diff
@@ -0,0 +1,251 @@
+    "base": "24",
+    "increment": "6"
+}
+````
+Which will result in the following response from Lambda:
+````
+{
+  "answer": "30",
+  "result": "success"
+}
+````
+
+### random_choice
+The random_choice operation returns a random selection from a provided list of options. For example, from the provided list of "hi,hello,yo,howdy,wassup" the function will select and return one option.
+#### Required Parameters
+The random_choice operation requires the following parameters:
+1. function = the operation to exectute, which should be set to `random_choice`
+2. list = the list of values you want to choose from. This should be a comma seperated list with no spaces.
+
+## Example
+````
+"Parameters": {
+    "function": "random_choice",
+    "list": "hi,hello,yo,howdy,wassup"
+}
+````
+Which will result in from Lambda similar to:
+````
+{
+  "answer": "yo",
+  "result": "success"
+}
+````
+
+### replace_text
+The replace_text repalces specifid text in an existing string with the supplied replacement.
+#### Required Parameters
+The replace_text operation requires the following parameters:
+1. function = the operation to exectute, which should be set to `replace_text`
+2. text_string = the text you are working with
+3. replace_this = the string in that text that you want to replace
+4. with_this = the string the you want to replace it with
+
+## Example
+````
+"Parameters": {
+    "function": "replace_text",
+    "text_string": "I don't like contractions.",
+    "replace_this": "don't",
+    "with_this": "do not"
+}
+````
+Which will result in the following response from Lambda:
+````
+{
+  "answer": "I do not like contractions.",
+  "result": "success"
+}
+````
+
+### split_text
+The split_text splits a provided string into two strings using a specified string as the split point. 
+#### Required Parameters
+The split_text operation requires the following parameters:
+1. function = the operation to exectute, which should be set to `split_text`
+2. text_string = the text you are working with
+3. split_at = the string of text that you want to use as your split point
+
+## Example
+````
+"Parameters": {
+    "function": "split_text",
+    "text_string": "User, Test",
+    "split_at": ", "
+}
+````
+Which will result in the following response from Lambda:
+````
+{
+  "segment_1": "User",
+  "segment_2": "Test",
+  "result": "success"
+}
+````
+
+### strip_text
+The strip_text strips text from the start, end, or from both sides of a string. It has three mode options: trim = strips text from the start and end of a string, left = strips only from the start of a string, right = strips only from the end of a string 
+#### Required Parameters
+The strip_text operation requires the following parameters:
+1. function = the operation to exectute, which should be set to `strip_text`
+2. mode = the mode you want to use. Options are: trim,left,right
+3. text_string = the text you are working with
+4. text_to_strip = the characters that you want to strip
```

Comment:
the example uses "strip_this" instead. Is this a regex? how is it matching?

Reply:
fixed. Its using the native strip functions of python. It simply strips any of the provided characters until it hits a non-specified character.